src/api/general.c

```diff
@@ -83,11 +85,6 @@ static const char nlopt_algorithm_names[NLOPT_NUM_ALGORITHMS][256] = {
     "Sequential Quadratic Programming (SQP) (local, derivative)",
     "CCSA (Conservative Convex Separable Approximations) with simple quadratic approximations (local, derivative)",
     "ESCH evolutionary strategy",
-#ifdef NLOPT_CXX11
-    "AGS (global, no-derivative)"
-#else
-    "AGS (NOT COMPILED)"
-#endif
```

Comment:
This PR broke `nlopt_algorithm_name` as it relied on this ordering that was changed...